src/components/button/index.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types'
+import './button.css'
+
+const Button = ({variant, children}) => {
+  return (
+    <button
+      className={`base ${
+        variant === 'fill' ? 'fill_button' : 'outline_button'
+      }`}
+    >
+      {children}
+    </button>
+  )
+}
+
+Button.propTypes = {
```

Comment:
The button component supports two variants 'fill' or 'outline'. The styling looks as mentioned here https://github.com/studybites/studybites/issues/3